test/src/test/java/org/corfudb/runtime/collections/PersistentCorfuTableTest.java

```diff
@@ -475,6 +478,41 @@ public void testUncommittedChangesIsolationBetweenParallelTxns() throws Exceptio
         assertThat(writerResult.get()).isEqualTo(payload2.getLsb());
     }
 
+    /**
+     * For MVO instances, the ObjectOpenOption.NO_CACHE should ensure that the instance
+     * is not saved in ObjectsView.objectCache
+     */
+    @Test
+    public void testNoCacheOption() {
+        addSingleServer(SERVERS.PORT_0);
+        rt = getNewRuntime(CorfuRuntime.CorfuRuntimeParameters.builder()
+                .maxCacheEntries(LARGE_CACHE_SIZE)
+                .build())
+                .parseConfigurationString(getDefaultConfigurationString())
+                .connect();
+        setupSerializer();
+
+        UUID streamA = UUID.randomUUID();
+        UUID streamB = UUID.randomUUID();
+
+        rt.getObjectsView()
+                .build()
+                .setStreamID(streamA)
+                .setTypeToken(new TypeToken<PersistentCorfuTable<String, String>>() {})
+                .option(ObjectOpenOption.CACHE)
+                .open();
+
+        rt.getObjectsView()
+                .build()
+                .setStreamID(streamB)
+                .setTypeToken(new TypeToken<PersistentCorfuTable<String, String>>() {})
+                .option(ObjectOpenOption.NO_CACHE)
+                .open();
```

Comment:
I don't think we should be supporting the NO_CACHE option at all for MVOs. Two disjoint MVOs for the same object in the same runtime will share the MVOCache, and if the time windows are different, they can interfere with one another in the cache if it's full.

Reply:
`NO_CACHE` is required by compactor to release a table from memory during checkpointing. We will see compactor OOM if this is not supported.